test/runner/tob.ts

```diff
@@ -1,5 +1,5 @@
 import { printSourceFile } from '../../src/printer';
-import { parseScript, parseModule } from '../../src/kataw';
+import { parseScript, parseModule, transformScript, transformModule } from '../../src/kataw';
```

Comment:
seems `transformScript, transformModule` was not found. did you forget to push?

Reply:
Look in the main branch. They are there, so maybe sync the brances?

Reply:
https://github.com/kataw/kataw/blob/main/src/kataw.ts#L198-L204